desktop-fonts/maple-font/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME="maple-font"
+PKGDES="Open source monospace font with round corner, ligatures and Nerd-Font for IDE and command line."
```

Comment:
Monospace font with round corner, ligatures, and nerd-font integration

Reply:
Note: No period points and no unhelpful advertisement.

Again, see the [Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/).